clarity_ext/context.py

```diff
@@ -196,6 +196,104 @@ def commit(self):
             obj.commit()
 
 
+class FakingEntityMonkey:
```

Comment:
Using the word monkey in a class that is solely doing monkey patching couldn't be that bad or?
and, I forgot to move the class...